services/metadata/src/metadata/routes/domain/avus.clj

```diff
@@ -24,7 +24,8 @@
    :created_by (describe String "The ID of the user who created the AVU")
    :modified_by (describe String "The ID of the user who last modified the AVU")
    :created_on (describe Long "The date the AVU was created in ms since the POSIX epoch")
-   :modified_on (describe Long "The date the AVU was last modified in ms since the POSIX epoch")})
+   :modified_on (describe Long "The date the AVU was last modified in ms since the POSIX epoch")
+   (s/optional-key :avus) (describe [(s/recursive #'Avu)] "AVUs attached to this AVU")})
```

Comment:
Yes! It's also how we're documenting our [data-info NavigationResponse](https://github.com/cyverse/DE/blob/2.6.0-RELEASE/services/data-info/src/data_info/routes/domain/navigation.clj#L12-L19) and [app tree parameters](https://github.com/cyverse/DE/blob/2.6.0-RELEASE/services/apps/src/apps/routes/domain/app.clj#L45-L51).